app/Http/Controllers/PrioController.php

```diff
@@ -0,0 +1,473 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\{AuditLog, Character, Guild, Instance, Item, Raid};
+use Auth;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class PrioController extends Controller
+{
+    const MAX_PRIOS = 10;

```

Comment:
I believe that this limit is a great idea to cut down on the amount of entries we are managing. 

But can we add some type of UI note around this? The only reason I found out we limited to 10 was because of this line. :)

Reply:
Also.. would it be useful to pull this back to a config setting? (Maybe in .env?)

Reply:
I've stuck 'max ###' into a number of places on the website, I can add it to the header for the first entry in the prio assignment page. Good call.

The only other way someone would find out is by filling out all 10 and not being able to add any more.

Regarding and environment variable, it's doable. But I don't think it's much of a priority unless we feel we need to be able to configure this variable at runtime.

Reply:
Env Var is me leaning more on being able to tweak things at run time vs having to make a code change.

Reply:
In that case, feel free to create a new issue to migrate constants over to config variables. This system _does_ lend itself to being more accessible for an open source project. The changes in said PR could attempt to load the config variable, and default to the constant if no variable is found.